ethereum/forge-test/GasBenchmark.t.sol

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import "forge-std/Test.sol";
+
+import "@pythnetwork/pyth-sdk-solidity/IPyth.sol";
+import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
+import "./utils/WormholeTestUtils.t.sol";
+import "./utils/PythTestUtils.t.sol";
+
+contract GasBenchmark is Test, WormholeTestUtils, PythTestUtils {
+    // 19, current mainnet number of guardians, is used to have gas estimates
+    // close to our mainnet transactions.
+    uint8 constant NO_GUARDIANS = 19;
+    uint8 constant NO_SIGNERS = 13;
```

Comment:
Do we know what the average number of signers real VAAs have is? I was under the impression it could be more than 13, but I could definitely be wrong.

Reply:
I asked around and it seems that it is possible to get more than 13, but the median seems to be 13. I'll add a comment to explain it too.